### DISCUSSION TITLE
Send later when Thunderbird or the pc is shut down

### DISCUSSION THREAD

**Discussion by GALASIO** [Extended-Thunder/send-later#701] [Q&A] [answered]
Is it possible that the extension can send emails later even when Thunderbird or the pc is shut down? For example, I plan an email the evening before, then I shut down my pc and in the next morning I would like the email to be send without booting my pc.

**Comment by Bluestone357** [comment]
No.  This is NOT possible.
The pc AND Thunderbird must be active.

**Comment by jikamens** [comment, marked as answer]
Please consult [the user guide](https://extended-thunder.github.io/send-later/#thunderbird-must-be-running-for-scheduled-messages-to-be-sent) before posting questions here, thanks. As noted there, your computer needs to be on and connected to the internet, and Thunderbird needs to be running, for messages to be sent at the scheduled time.
